### PR TITLE
feat(modal): 增加反馈弹窗modal组件

### PR DESCRIPTION
**Pull request by MNZhu**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/ui-vue/blob/main/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/modal/vars.less [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+.component-css-vars-modal() {
+  --ti-mobile-modal-text-font-size: 14px;
+  --ti-mobile-modal-header-font-size: 16px;
+  --ti-mobile-modal-text-color: var(--ti-mobile-common-color-text-primary, #191919);
+  --ti-mobile-modal-alert-font-size: 22px;
+  --ti-mobile-modal-box-bg-color: #fff;
+  --ti-mobile-modal-box-shadow: 0 8px 40px 0 rgba(0, 0, 0, 0.2);
+  --ti-mobile-modal-close-btn-top: 13px;
+  --ti-mobile-modal-close-btn-scale: scale(1, 1);
+  --ti-mobile-modal-close-btn-padding-horizontal-vertical: 0;
```
Comment (chenxi-20) [inline src/modal/vars.less]: padding需要拆开,并且长度已经超了50

Comment (MNZhu) [reply src/modal/vars.less]: 已删除无意义的变量定义

File: src/modal/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * A PARTICULAR PURPOSE. SEE THE APPLICABLE LICENSES FOR MORE DETAILS.
+ *
+ */
+
+export default {
+  "tiny-mobile-modal-text-font-size": "14px",
+  "tiny-mobile-modal-header-font-size": "16px",
+  "tiny-mobile-modal-text-color": "#191919",
+  "tiny-mobile-modal-alert-font-size": "22px",
+  "tiny-mobile-modal-box-background-color": "#fff",
```
Comment (chenxi-20) [inline src/modal/index.js]: background-color没有对应vars.less里面的bg-color，下面几个也有类似的问题

File: src/modal/vars.less [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+.component-css-vars-modal() {
+  --ti-mobile-modal-text-font-size: 14px;
+  --ti-mobile-modal-header-font-size: 16px;
+  --ti-mobile-modal-text-color: var(--ti-mobile-common-color-text-primary, #191919);
+  --ti-mobile-modal-alert-font-size: 22px;
+  --ti-mobile-modal-box-bg-color: #fff;
+  --ti-mobile-modal-box-shadow: 0 8px 40px 0 rgba(0, 0, 0, 0.2);
+  --ti-mobile-modal-close-btn-top: 13px;
+  --ti-mobile-modal-close-btn-scale: scale(1, 1);
+  --ti-mobile-modal-close-btn-bg-color-hover: #fff;
```
Comment (zzcr) [inline src/modal/vars.less]: 这里大部分的颜色最好都提成公共变量

File: src/modal/index.js (hunk 2 above)
Comment (MNZhu) [reply src/modal/index.js]: 已完成整改

File: src/modal/vars.less (hunk 3 above)
Comment (MNZhu) [reply src/modal/vars.less]: 已提取整改

**Review by zzcr — APPROVED** [review]

**Pull request merged**